tests/controltab/test_tab_alarm_warn.py

```diff
@@ -81,6 +82,9 @@ def test_callback_selection_changed(qtbot, widget):
         pos=center,
     )
 
+    # Sleep one second to let the event loop to have the time to run the signal
```

Comment:
Better replace with:
```
Sleep so the event loop can access CPU to handle the signal
```

If replaced, please replace all the occurrences, not only that one.

Reply:
Updated in the commit: [7d7ab7f](https://github.com/lsst-ts/ts_m2gui/pull/17/commits/7d7ab7f36a1d11880485b59c021ba2289c02f003). Thanks!